rest/data/project_settings.go

```diff
@@ -412,7 +412,14 @@ func SaveProjectSettingsForSection(ctx context.Context, projectId string, change
 		if catcher.HasErrors() {
 			return nil, errors.Wrap(catcher.Resolve(), "invalid project trigger")
 		}
+	case model.ProjectPageViewsAndFiltersSection:
+		err = model.ValidateParsleyFilters(mergedSection.ParsleyFilters)
+		catcher.Add(err)
```

Comment:
Since you aren't looping, there's not actually a reason to use the catcher here, you can just do a regular error check